packages/neutrino-preset-web/index.js

```diff
@@ -99,6 +99,10 @@ module.exports = (neutrino) => {
       .chunkFilename('[id].[chunkhash].js')
       .end()
     .resolve
+    .alias
+        // TIP: make sure 2 version of "core-js" always match in package.json and babel-polyfill/package.json
+        .set('core-js', join(require.resolve('core-js'), '..'))
```

Comment:
Instead of a join with `..`, let's use `dirname` from path:

```js
const { join, dirname } = require('path');

// ...

.set('core-js', dirname(require.resolve('core-js')))
```